morebits.js

```diff
@@ -1241,7 +1241,7 @@ Morebits.unbinder.prototype = {
 	 * @param {string} postfix
 	 */
 	unbind: function UnbinderUnbind( prefix, postfix ) {
-		var re = new RegExp( prefix + '(.*?)' + postfix, 'gs' );
+		var re = new RegExp( prefix + '([\s\S]*?)' + postfix, 'g' ); //eslint-disable-line no-useless-escape
```

Comment:
lint is whining appropriately here, because this won't do what you think it will.  When using the contructor format instead of the regex literal, you have to escape the escape.
```suggestion
		var re = new RegExp( prefix + '([\\s\\S]*?)' + postfix, 'g' );
```

Reply:
Right! I hastily tested w/o using regex constructor form and decided lint was wrong.